### PR TITLE
Added docs for publishing a buildpackage to the buildpack registry

### PR DESCRIPTION
**Pull request by jkutner**
Documents the publish process for the [Buildpack Registry](https://github.com/buildpacks/spec/blob/main/extensions/buildpack-registry.md)

### CODE REVIEW

File: content/docs/buildpack-author-guide/publish-a-buildpack.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+### 2. Managing your namespace
+
+The first time you publish a buildpack with a given namespace, the registry will automatically assign your GitHub user as that namespace's owner. From then on, only you can publish new buildpacks or buildpack versions under that namespace.
+
+If you try to publish a buildpack with a namespace that's already in use, the request will fail and the GitHub issue will be closed.
+
+You can add or change namespace owners by submitting a Pull Request to the [buildpacks/registry-namespaces](https://github.com/buildpacks/registry-namespaces/).
+
+### 3. Automating the registration
```
Comment (jromero) [inline content/docs/buildpack-author-guide/publish-a-buildpack.md:44]: This is really valuable information but I'm not sure it should be labeled as a "step" given that A) it's more of an optional advanced setup B) they may choose to automate it via another CI/CD tool.

My proposal here would be to make it a more distinct section within this page or as a separate page altogether named "Automating the registration via GitHub Actions"

File: content/docs/buildpack-author-guide/publish-a-buildpack.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Click _Submit new issue_, and your request will be processed within seconds. If the image is a valid buildpack, it will be added to the registry. If there is a problem, the issue will be tagged as a "Failure" and a comment will be added with a link to get more details. Whether successful or not, the issue will be closed.
+
+### 2. Managing your namespace
+
+The first time you publish a buildpack with a given namespace, the registry will automatically assign your GitHub user as that namespace's owner. From then on, only you can publish new buildpacks or buildpack versions under that namespace.
+
+If you try to publish a buildpack with a namespace that's already in use, the request will fail and the GitHub issue will be closed.
+
+You can add or change namespace owners by submitting a Pull Request to the [buildpacks/registry-namespaces](https://github.com/buildpacks/registry-namespaces/).
```
Comment (jromero) [inline content/docs/buildpack-author-guide/publish-a-buildpack.md:42]: This doesn't sound like a "step" either but instead maybe more informational.

```suggestion
> **Managing your namespace**
> 
> The first time you publish a buildpack with a given namespace, the registry will automatically assign your GitHub user as that namespace's owner. From then on, only you can publish new buildpacks or buildpack versions under that namespace.
> 
> If you try to publish a buildpack with a namespace that's already in use, the request will fail and the GitHub issue will be closed.
> 
> You can add or change namespace owners by submitting a Pull Request to the [buildpacks/registry-namespaces](https://github.com/buildpacks/registry-namespaces/).
```

File: content/docs/buildpack-author-guide/publish-a-buildpack.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+Make sure you've followed the steps in [Package a buildpack][package] to create your buildpack image and publish it to an OCI registry.
+
+**NOTE:** In order to publish your buildpack to the registry, its `id` must be in the format `<namespace>/<name>`. For example:
+
+```toml
+[buildpack]
+id = "example/my-cnb"
+```
+
+### 1. Running the CLI command
```
Comment (jromero) [inline content/docs/buildpack-author-guide/publish-a-buildpack.md:20]:
```suggestion
### 1. Register your buildpack
```

File: content/docs/buildpack-author-guide/publish-a-buildpack.md (hunk 1 above)
Comment (jkutner) [reply content/docs/buildpack-author-guide/publish-a-buildpack.md:44]: yea, similar feels, but I'll look into making this its own page

Comment (jkutner) [reply content/docs/buildpack-author-guide/publish-a-buildpack.md:44]: Another option is to omit this from the docs, and relegate it to a blog post only.

Comment (jkutner) [reply content/docs/buildpack-author-guide/publish-a-buildpack.md]: Address in c55f768

**Review by jromero — APPROVED** [review]
Looks great! Approved! 

Don't know if this is tied to the fanfare of the Buildpack Registry release so I'll leave it up to you to merge when deemed appropriate.

File: content/docs/buildpack-author-guide/publish-a-buildpack.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+```shell script
+pack buildpack register example/my-cnb
+```
+
+This will open GitHub in a browser and may ask you to authenticate with GitHub. After doing so, you'll see a pre-populated GitHub Issue with the details of your buildpack. For example:
+
+<img src="/images/registry-add-buildpack.png" />
+
+The pre-populated text in the body of the issue is consider structured data, and will be used to automatically add the buildpack to the registry index. Do not change it.
```
Comment (natalieparellano) [inline content/docs/buildpack-author-guide/publish-a-buildpack.md:32]:
```suggestion
The pre-populated text in the body of the issue is considered structured data, and will be used to automatically add the buildpack to the registry index. Do not change it.
```

**Pull request merged**